.travis.yml

```diff
@@ -1,162 +1,63 @@
-  - export CC=${CC_COMPILER}
-  - export CXX=${CXX_COMPILER}
-  - JOBS=2 # Travis machines have 2 cores.
-  - |
-    # getting Catch2 library
-    git clone https://github.com/catchorg/Catch2.git
-    cd Catch2
-    git checkout v2.9.1
-    mkdir build
-    cd build
-    pwd
-    cmake -DCATCH_BUILD_TESTING=OFF -DCATCH_INSTALL_DOCS=OFF -DCATCH_INSTALL_HELPERS=OFF -DCMAKE_INSTALL_PREFIX=../installed ..
-    cmake --build . --target install --config Release
-    cd ../..
+      - <<: *osx
+        osx_image: xcode9.4
+        env: CONAN_APPLE_CLANG_VERSIONS=9.1
 
-    # If linux and clang install the right version of libc++.
-    if [[ "${TRAVIS_OS_NAME}" == "linux" && "${CXX%%+*}" == "clang" && -n "$(ls -A ${LLVM_INSTALL})" ]]; then
-      LLVM_INSTALL=${DEPS_DIR}/llvm/install
-      if   [[ "${CXX}" == "clang++-3.5" ]]; then LLVM_VERSION="3.5.2";
-      elif [[ "${CXX}" == "clang++-3.6" ]]; then LLVM_VERSION="3.6.2";
-      elif [[ "${CXX}" == "clang++-3.7" ]]; then LLVM_VERSION="3.7.1";
-      elif [[ "${CXX}" == "clang++-3.8" ]]; then LLVM_VERSION="3.8.1";
-      elif [[ "${CXX}" == "clang++-3.9" ]]; then LLVM_VERSION="3.9.1";
-      elif [[ "${CXX}" == "clang++-4.0" ]]; then LLVM_VERSION="4.0.1";
-      elif [[ "${CXX}" == "clang++-5.0" ]]; then LLVM_VERSION="5.0.2";
-      elif [[ "${CXX}" == "clang++-6.0" ]]; then LLVM_VERSION="6.0.1";
-      elif [[ "${CXX}" == "clang++-7" ]]; then LLVM_VERSION="7.0.1";
-      elif [[ "${CXX}" == "clang++-8" ]]; then LLVM_VERSION="8.0.0";
-      fi
-      LLVM_URL="http://llvm.org/releases/${LLVM_VERSION}/llvm-${LLVM_VERSION}.src.tar.xz"
-      LIBCXX_URL="http://llvm.org/releases/${LLVM_VERSION}/libcxx-${LLVM_VERSION}.src.tar.xz"
-      LIBCXXABI_URL="http://llvm.org/releases/${LLVM_VERSION}/libcxxabi-${LLVM_VERSION}.src.tar.xz"
-      mkdir -p llvm llvm/build llvm/projects/libcxx llvm/projects/libcxxabi
-      travis_retry wget -O - ${LLVM_URL} | tar --strip-components=1 -xJ -C llvm
-      travis_retry wget -O - ${LIBCXX_URL} | tar --strip-components=1 -xJ -C llvm/projects/libcxx
-      travis_retry wget -O - ${LIBCXXABI_URL} | tar --strip-components=1 -xJ -C llvm/projects/libcxxabi
-      (cd llvm/build && cmake .. -DCMAKE_INSTALL_PREFIX=${LLVM_INSTALL})
-      (cd llvm/build/projects/libcxx && sudo make install -j${JOBS})
-      (cd llvm/build/projects/libcxxabi && sudo make install -j${JOBS})
-      export CXXFLAGS="-isystem ${LLVM_INSTALL}/include/c++/v1"
-      export LDFLAGS="-L ${LLVM_INSTALL}/lib -l c++ -l c++abi"
-      export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${LLVM_INSTALL}/lib"
-    fi
+      - <<: *osx
+        osx_image: xcode10.1
+        env: CONAN_APPLE_CLANG_VERSIONS=10.0
 
-before_script:
-  - rm -rf build
-  - mkdir -p build
-  - cd build
-  - pwd
-  - cmake -G "Unix Makefiles" -DCMAKE_PREFIX_PATH=../Catch2/installed ..
+install:
+  - chmod +x .ci/install.sh
+  - ./.ci/install.sh
 
 script:
-  - cmake --build . -- -j${JOBS}
-  - ctest --output-on-failure -j${JOBS}
-
-notifications:
```

Comment:
Need return
`notifications:
  email: false`
To disable email notif.

Reply:
That's a very good idea :D

Reply:
I found how disable mail in settings on site.

Reply:
I would prefer to set everything in the `yml` config file if that is not security-critical.
I think it's better to track setting changes via git as well.

So, in my opinion, this should be here as well.

Reply:
Fine, I also like it when I can configure different notifications for different repositories.